search-app/src/cmr/search/services/autocomplete_service.clj

```diff
@@ -65,19 +65,24 @@
       (and (seq token) (seq types)) (non-empty-token-with-type root types group-ids)
       (and (seq token) (empty? types)) (non-empty-token-without-type root group-ids))))
 
-(defn autocomplete
-  "Execute elasticsearch query to get autocomplete suggestions"
-  [context term types opts token]
-  (let [acls (af/get-acls context [:catalog-item])
-        group-permissions (map :group-permissions acls)
+(defn fetch-group-ids-for-autocomplete
```

Comment:
change this to `(defn- fetch-group-ids-for-autocomplete`

Reply:
This needs to be public so that it can be unit tested